resbaz/resbazTucson2021.md

```diff
@@ -17,7 +17,7 @@ May 17&ndash;21, 2021
 
 ## Schedule
 
-Note: Due to the "unconference" nature of ResBaz, you can opt to teach a workshop during the [registration](#register) process. Therefore, the schedule will not be final until the day of! Below represents the current schedule.
+Note: Due to the "unconference" nature of ResBaz, you can opt to teach a workshop during the [registration](#register) process. Therefore, the schedule will not be final until the day of! Below represents the current schedule. Click [here](https://kelseygonzalez.github.io/resbaz2021schedule/) to view the schedule in a new window.
 
 <div style="width:100%;overflow-x:auto">
 <iframe src="https://kelseygonzalez.github.io/resbaz2021schedule/" width="830" height="650" scrolling="auto" frameborder="no"></iframe>
```

Comment:
```suggestion
<iframe src="https://kelseygonzalez.github.io/resbaz2021schedule/" width="830" height="750" scrolling="auto" frameborder="no"></iframe>
```

Sorry, meant to get to this over the weekend. This should add a little more vertical space so less is clipped naturally.